schemas/arb.json

```diff
@@ -361,10 +365,14 @@
         "@@author": {
             "description": "The author of these messages. In the case of localized ARB files it can contain the names/details of the translator.",
             "type": "string"
+        },
+        "@@comment": {
+            "description": "A top-level comment for this file.",
```

Comment:
Might also be useful to indicate the audience of `@@comment`, i.e. it is not intended for translators.